core/ir/opnd_api.h

```diff
@@ -1695,6 +1695,11 @@ typedef enum _dr_opnd_flags_t {
     DR_OPND_EXTENDED = 0x20,
     /** This immediate integer operand should be interpreted as an AArch64 extend type. */
     DR_OPND_IS_EXTEND = 0x40,
+    /** Registers with this flag should be considered vectors and have a subsize
```

Comment:
style: https://dynamorio.org/page_code_style.html#autotoc_md124 says "For comments starting with /∗∗, leave the rest of the first line empty, unless the entire comment is a single line."

Reply:
done